.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,83 @@
+name: build-pr
+
+on:
```

Comment:
If we build the PR on any tags/and on `master` it would save us (usually @felixarntz) having to build the production ZIP by hand on a Mac. Having the CI generate our builds instead of us doing it manually is way nicer, so I filed #1196 to chat about that and extend this later 😄

Reply:
Good call!